client/db/bolt/db.go

```diff
@@ -1114,36 +1103,42 @@ func (db *BoltDB) UpdateOrderMetaData(oid order.OrderID, md *dexdb.OrderMetaData
 			return fmt.Errorf("UpdateOrderMetaData: %w", err)
 		}
 
-		var linkedB []byte
-		if !md.LinkedOrder.IsZero() {
-			linkedB = md.LinkedOrder[:]
-		}
+		return updateOrderMetaData(oBkt, md)
+	})
+}
 
-		var accelerationsB encode.BuildyBytes
-		if len(md.AccelerationCoins) > 0 {
-			accelerationsB = encode.BuildyBytes{0}
-			for _, acceleration := range md.AccelerationCoins {
-				accelerationsB = accelerationsB.AddData(acceleration)
-			}
+func updateOrderMetaData(bkt *bbolt.Bucket, md *dexdb.OrderMetaData) error {
+	var linkedB []byte
+	if !md.LinkedOrder.IsZero() {
+		linkedB = md.LinkedOrder[:]
+	}
+
+	var accelerationsB encode.BuildyBytes
+	if len(md.AccelerationCoins) > 0 {
+		accelerationsB = encode.BuildyBytes{0}
+		for _, acceleration := range md.AccelerationCoins {
+			accelerationsB = accelerationsB.AddData(acceleration)
 		}
+	}
 
-		return newBucketPutter(oBkt).
-			put(statusKey, uint16Bytes(uint16(md.Status))).
-			put(updateTimeKey, uint64Bytes(timeNow())).
-			put(proofKey, md.Proof.Encode()).
-			put(changeKey, md.ChangeCoin).
-			put(linkedKey, linkedB).
-			put(swapFeesKey, uint64Bytes(md.SwapFeesPaid)).
-			put(maxFeeRateKey, uint64Bytes(md.MaxFeeRate)).
-			put(redemptionFeesKey, uint64Bytes(md.RedemptionFeesPaid)).
-			put(fromVersionKey, uint32Bytes(md.FromVersion)).
-			put(toVersionKey, uint32Bytes(md.ToVersion)).
-			put(optionsKey, config.Data(md.Options)).
-			put(redemptionReservesKey, uint64Bytes(md.RedemptionReserves)).
-			put(refundReservesKey, uint64Bytes(md.RefundReserves)).
-			put(accelerationsKey, accelerationsB).
-			err()
-	})
+	return newBucketPutter(bkt).
+		put(statusKey, uint16Bytes(uint16(md.Status))).
+		put(updateTimeKey, uint64Bytes(timeNow())).
+		put(proofKey, md.Proof.Encode()).
+		put(changeKey, md.ChangeCoin).
+		put(linkedKey, linkedB).
+		put(swapFeesKey, uint64Bytes(md.SwapFeesPaid)).
+		put(maxFeeRateKey, uint64Bytes(md.MaxFeeRate)).
+		put(redemptionFeesKey, uint64Bytes(md.RedemptionFeesPaid)).
+		put(fromVersionKey, uint32Bytes(md.FromVersion)).
+		put(toVersionKey, uint32Bytes(md.ToVersion)).
```

Comment:
I think at least the two max fee rates and to/from versions are immutable and thus best suited for teh `put`s back in `UpdateOrder`.